core/os/device/device.proto

```diff
@@ -193,7 +193,7 @@ message GPUProfiling {
   bool hasRenderStage = 1;
   // The GPU performance counters.
   GpuCounterDescriptor gpu_counter_descriptor = 2;
-  reserved 3;
```

Comment:
You shouldn't re-use reserved ones. They are reserved because they used to be used for something else. This could cause deserialization of old protos fail. While this isn't technically an issue with this proto, we shouldn't re-use reserved ones.

Reply:
Done.